aerogearcatalog/unifiedpush-apb
# fix service error and make apb re-deployable

Fixes a problem with an error displayed in the Service instance even though the provision was successful. Also fixes a problem where some resources were not properly cleaned up on deletion. Now you should be able to delete the apb and re-deploy it to the same namespace.

Verification:

1. Check out this branch
1. Build and push the apb to your Openshift instance
1. Deploy the apb to a project. It should succeed and no error should be shown in the service instance.
1. Delete the apb via the service instance and redeploy it.
1. This should also work.
